src/Sylius/Bundle/SettingsBundle/Manager/SettingsManager.php

```diff
@@ -134,7 +118,6 @@ public function loadSettings($namespace)
 
     /**
      * {@inheritdoc}
-     * @throws ValidatorException
```

Comment:
IIRC this is correct, cause this method throws exception that is not listed in interface.